AppServer/google/appengine/api/taskqueue/taskqueue.py

```diff
@@ -864,17 +865,17 @@ def __host_from_target(target):
     """
     default_hostname = app_identity.get_default_version_hostname()
     if default_hostname is None:
-
-
-
       return None
+    # AppScale: We do not support the instance parameter.
+    target_info = target.split('.')
+    if len(target_info) > 2:
+      raise InvalidTaskError(
```

Comment:
I'm a little confused about this one, the target is a property of the task and is determined here by resolve_hostname_and_target. It's not technically its own header so the request object doesn't contain information about whether we have a target or not.